src/components/donation/Steps/DonateMethods/DonateMethods.tsx

```diff
@@ -49,27 +46,7 @@ export default function DonateMethods({ donaterConfig, state }: Props) {
           />
         </Tab.Panel>
         <Tab.Panel className="grid">
-          {!wallet ? (
```

Comment:
Wallet is not needed anymore to show donate form 
<img width="611" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/05f0f484-3365-4d4a-b4ee-d21642c30f4f">

balances, chain mismatch check, simulation are handled just before submission